StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentMethodFormViewController.swift

```diff
@@ -288,11 +288,15 @@ extension PaymentMethodFormViewController {
                 self.delegate?.updateErrorLabel(for: genericError)
             }
         }
+        let additionalParameters: [String: Any] = [
+            "hosted_surface": "payment_element",
+        ]
```

Comment:
similar pattern/precedent of passing parameters already exists here:

https://github.com/stripe/stripe-ios/blob/e371135c/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkPaymentController.swift#L135-L138